packages/inventory/src/Stocktake/api/hooks.ts

```diff
@@ -28,29 +29,29 @@ import {
 
 export const useStocktakeApi = (): StocktakeApi => {
   const { client } = useOmSupplyApi();
-  return getSdk(client);
+  const { store } = useHostContext();
+  const queries = getStocktakeQueries(getSdk(client), store.id);
+  return { ...queries, storeId: store.id };
```

Comment:
I think this is nicer. Though, I would also like to put query keys here to reduce possible mistakes through the duplication we currently have.. something like `detailKey`, `listKey` 🤔

Reply:
oh, that is good.